tests/envs/test_cover.py

```diff
@@ -339,22 +314,32 @@ def test_cover_multistep_options():
         # Ungrasp
         np.array([0., 0., -0.1], dtype=np.float32),
     ]
-    make_video = False  # Can toggle to true for debugging
 
-    def policy(s: State) -> Action:
-        del s  # unused
-        return Action(action_arrs.pop(0))
+    policy = utils.action_arrs_to_policy(action_arrs)
+
+    # Here's an example of how to make a video within this test.
```

Comment:
So we're making it harder to generate the videos here right? If someone wants to generate videos for one of the other tests in this file, they would have to copy paste this example, and figure out what arguments to change. What's the reason to move away from what we had before, where we could just use monitor = either None or utils.VideoMonitor(env.render)?

Reply:
what we had before would get messy because we would have a bunch of commented out lines for actually saving the video to a file. I think this is a fine compromise, I don't think we're going to be making new videos in unit tests too frequently, but when we do, we can refer to this example to start.